.github/workflows/on-pr.yaml

```
@@ -6,16 +6,29 @@ jobs:
   lifecycle-install-tests:
     name: Test, Build & Push
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: read
     steps:
     - name: Checkout code
       uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)